src/pages/Home/index.js

```diff
@@ -4,17 +4,18 @@ import Header from "../../components/Header";
 import Kurser from "../../components/Kurser";
 import Partners from "../../components/Partners";
 import PartnerLogos from "../../components/partnersLogos";
-
+import Events from "../../components/Events";
 import "./home.css";
 function index() {
 	return (
 		<>
 			<div className="homepage-container">
-			  <Header />
-      <Partners />
-      <Kurser />
```

Comment:
Nicely done. I would only suggest that we stick to one same language when we name our components.

Reply:
Has a good design but ux designers they are changing the design before their presentation.